### ISSUE TITLE
Documentation theming

### ISSUE THREAD

**Issue by jeff-h** [MacGapProject/documentation#2]
@tschundeee — I am wondering if you'd be willing to apply a bit of your magic to the CSS / HTML of the docs site? It would be good to tie it in with your main site.

**Comment by PatrikTheDev** [comment]
Hi, if you would like some help I would like to help you, some modern design always helps😉 btw the site is broken on mobile

**Comment by jeff-h** [comment]
Sounds great! What would you need from me to move ahead on this?

**Comment by PatrikTheDev** [comment]
I am on vacation now, when I will be home I would start, should I fork it or you would give me access to this repo? If you would give me an donation I would be really happy😉.

**Comment by jeff-h** [comment]
Fork it for now and we can move on from there.

The biggest problem with MacGap development is, I think, that those with the knowledge to contribute to the code don't really need or use the project (if you can write Obj-C, you don't really need to write a native-style app in Javascript). So unfortunately there's not much money involved. Well actually none :)

**Comment by PatrikTheDev** [comment]
You have PR on the landing page, that is done, if you like it I will get to work and will continue on the documentation, if you will see an problem just tell me, it is really fresh site, I started working on it like 2 hours ago

**Comment by chriskol** [comment]
@Patrik-svobodik  Can you link the PR?  I can't seem to see it on the repo :(

Edit: Nevermind, I [see it now](https://github.com/MacGapProject/MacGapProject.github.io/pull/7), it's in the [other repo](https://github.com/MacGapProject/MacGapProject.github.io)!

**Comment by PatrikTheDev** [comment]
What do you think?

**Comment by jeff-h** [comment]
Sorry for the delay — any way I can preview this prior to rolling it in? Thanks for your work!

**Comment by jeff-h** [comment]
Ah, I realised I could just save it as html and open it that way.

**Comment by gobijan** [comment]
I'll have a look too :)

**Comment by gobijan** [comment]
I think this looks by far not finished...
![bildschirmfoto 2018-08-16 um 11 37 31](https://user-images.githubusercontent.com/49339/44201247-12821680-a149-11e8-9b20-1de8abc10d36.png)

**Comment by PatrikTheDev** [comment]
You are right, it is not fully finished, but the Material design is chosen
because I like the style, and the messy code of the old website (it was an
theme so it is hard to work with it), why do you think it is “by far not
finished”?
čt 16. 8. 2018 v 11:39 odesílatel Bijan Rahnema <notifications@github.com>
napsal:

**Comment by PatrikTheDev** [comment]
The itemX are a mistake, I uploaded an older version of the HTML to github
čt 16. 8. 2018 v 17:45 odesílatel Patrik Svoboda <svob.pata@gmail.com>
napsal:

**Comment by jeff-h** [comment]
@Patrik-svobodik Thanks for your work. I agree that a Material Design foundation could give the site a more modern appearance than the current design. I'd like to see your version modified with a more cohesive colour scheme. Would you be able to try that? I'd also suggest removing the darker boxes entirely, and perhaps putting some of the content side-by-side in three columns.

@chriskol how would you feel about us using screenshots of your apps on this page, as examples of what's possible?

**Comment by PatrikTheDev** [comment]
OK, what colors would you like more? I will do the columns

pá 17. 8. 2018 v 12:46 odesílatel jeff-h <notifications@github.com> napsal:

**Comment by jeff-h** [comment]
I'm happy for you to pick a colour scheme you like :) Perhaps check out these sites for inspiration: 

* https://coolors.co
* http://colormind.io

**Comment by PatrikTheDev** [comment]
Hi, I just found this site’s files on my disk and realised it is not finished😃 I will work on it when I get home (should be soon)
